### DISCUSSION TITLE
Unclear how to invite users

### DISCUSSION THREAD

**Discussion by maximo1984** [Kareadita/Kavita#3263] [Ideas]
### Idea Description

I was able to create a username and password in the register user page, but what link do I give them to join?

### Idea Category

User Experience

### Duration of Using Kavita

6 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
This should be a bug for support. 

After you create an account via the Invite flow, Kavita will give you a link to give them unless you already setup the SMTP settings, then it will email them out a link to setup their account on your server. 

This is spelled out in the UI as well. 

This is not a feature request, so either create a bug or reach out in the discord to receive more help.

**Comment by maximo1984** [comment]
When I tried to invite via email, it says "Notice: Email on file is not a valid email and can not be sent. A link has been dumped in logs. The admin can provide this link to complete flow. " However, the link provided is a localhost address

**Reply by majora2007** [reply]
So when you set up the user, you didn't put in a real email, so it can't email the user. Then it's saying hey here's a link you can send them, but of course it's going to use localhost because it doesn't know what your reverse proxy is. 

What you need to do is use a real email and set The host name in the settings. This will allow for your link to have your reverse proxy and when you use a real email, it'll just send an email to them, assuming you have the email settings set.

**Comment by maximo1984** [comment]
It is a real email though. And where do I setup reverse proxy? The instructions for all of this are very unclear.

**Reply by majora2007** [reply]
Please dm me/email me the url it dumps so I can understand more. If you're on discord that will be the easiest, if not, use the email on my profile/commits.

**Comment by maximo1984** [comment]
I found the reverse proxy setup on Wiki, i think I can go from there. But will join discord.
